web/src/messages_overlay_ui.ts

```diff
@@ -28,13 +40,17 @@ export function focus_on_sibling_element(context) {
         elem_to_be_focused_id = $prev_row.attr(context.id_attribute_name);
     }
 
-    const new_focus_element = get_element_by_id(elem_to_be_focused_id, context);
+    const new_focus_element = get_element_by_id(elem_to_be_focused_id ?? "", context);
     if (new_focus_element[0] !== undefined) {
-        activate_element(new_focus_element[0].children[0], context);
+        activate_element(new_focus_element[0].children[0] as HTMLElement, context);
     }
 }
 
-export function modals_handle_events(_e, event_key, context) {
+export function modals_handle_events(
+    _e: JQuery.KeyDownEvent | JQuery.KeyPressEvent,
```

Comment:
`_e` has been unused since 4a3211d6af137c3cb6e55d730a60480792414de3. We should remove this in a prep refactoring commit.